zilencer/views.py

```diff
@@ -542,6 +542,7 @@ def update_remote_realm_data_for_server(
             host=realm.host,
             realm_deactivated=realm.deactivated,
             realm_date_created=timestamp_to_datetime(realm.date_created),
+            org_type=realm.org_type,
```

Comment:
Lower down we have the syncing logic for already existing `RemoteRealm` registrations. Do we want to sync `org_type` there, like the `deactivated` status, in case it changes? I could see us wanting to just keep the original value potentially :thinking:

Reply:
I think probably syncing changes is reasonable as long as we're storing audit log entries when it does.

Reply:
@timabbott Oops, I think I wasn't clear - the current behavior, as merged, will **not** sync `org_type` on existing `RemoteRealm`s if it changes in the remote server's `Realm`. I can create an extra commit to make it get synced - if we want.

It should be just a tiny change like this + ideally a test:
```
diff --git a/zilencer/views.py b/zilencer/views.py
index 324ec7b085..9415334fad 100644
--- a/zilencer/views.py
+++ b/zilencer/views.py
@@ -566,6 +566,7 @@ def update_remote_realm_data_for_server(
         for remote_realm_attr, realm_dict_key in [
             ("host", "host"),
             ("realm_deactivated", "deactivated"),
+            ("org_type", "org_type"),
         ]:
             old_value = getattr(remote_realm, remote_realm_attr)
             new_value = getattr(realm, realm_dict_key)
```

Reply:
I think it's not a priority; maybe plan to add a commit when we're doing `name`, since you'll be in the tests for it anyway?

Reply:
I think syncing it is probably good; we store the old values in `RemoteRealmAuditLog`, so original values will always be available.